datapipelines/src/main/kotlin/io/customer/datapipelines/plugins/ContextPlugin.kt

```diff
@@ -0,0 +1,33 @@
+package io.customer.datapipelines.plugins
+
+import com.segment.analytics.kotlin.core.Analytics
+import com.segment.analytics.kotlin.core.BaseEvent
+import com.segment.analytics.kotlin.core.platform.Plugin
+import com.segment.analytics.kotlin.core.utilities.putInContextUnderKey
+import io.customer.sdk.data.model.CustomAttributes
+
+/**
+ * Plugin class responsible for updating the context properties in events
+ * tracked by Customer.io SDK.
+ */
+class ContextPlugin(override var analytics: Analytics) : Plugin {
```

Comment:
we dont need to add `analytics` here as it would be added and set by timeline itself.